app/.env.test

```diff
@@ -1,3 +1,3 @@
 # Tests don't actually hit this endpoint, but we
 # need the environment variable to be defined.
```

Comment:
Seems worth updating this comment if the value is now important (or at least its format)